### PR TITLE
add multiple compat tools

### PR DESCRIPTION
**Pull request by NotAShelf**
- add following titanfall utilities
	- viper
	- flightcore
	- northstarproton

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,30 +7,33 @@ See an overview of the flake outputs by running
...
-[`winestreamproxy`](./pkgs/winestreamproxy)                 | Wine-Discord RPC (broken)
-[`proton-ge`](./pkgs/proton-ge)                             | Custom build of Proton with the most recent bleeding-edge Proton Experimental WINE
-
-* `legendaryBuilder` is a function that installs games with `legendary-gl`. You
-are expected to log in before using it, with `legendary auth`.
-The function takes an attrset containing at least the attrset `games` which
-includes the games you want installed. Optionally, you can set an `opts`
-attrset that will set the options you set inside for all games listed.
-You can find a usage example in [example.nix](./example.nix).
+| Package                                                     | Description                                                                                            |
```
Comment (fufexan) [inline README.md:10]: Can you remove the prefixing/suffixing `|`? They'll be annoying to change everytime a longer description is added.

File: pkgs/titanfall/northstar-proton.nix [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    buildInputs = [
+      gst_all_1.gst-plugins-base
+      libmspack
+      libpcap
+      libpressureaudio
+      libudev-zero
+      libusb1
+      ocl-icd
+    ];
+  }
```
Comment (fufexan) [inline pkgs/titanfall/northstar-proton.nix:44]: `meta`?

File: README.md (hunk 1 above)
Comment (NotAShelf) [reply README.md:10]: Could I ask you to do it? My editor does it instantly thanks to prettier

File: pkgs/titanfall/northstar-proton.nix (hunk 2 above)
Comment (NotAShelf) [reply pkgs/titanfall/northstar-proton.nix:44]: whoops.

**Review by fufexan — APPROVED** [review]

**Review by NotAShelf — COMMENTED** [review]
LGTM tbh

**Comment by fufexan** [comment]
image

**Pull request merged**